src/modules/createDao/components/createProcessForm/createProcessFormProposalCreation/proposalCreationSettingsDefault.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+
+import safeWallet from '@/assets/images/safeWallet.png';
+import type { ISetupBodyForm } from '@/modules/createDao/dialogs/setupBodyDialog';
+import type { IPluginProposalCreationSettingsParams } from '@/modules/createDao/types';
+import { useFormField } from '@/shared/hooks/useFormField';
+import { addressUtils, CheckboxCard, type CheckboxState } from '@aragon/gov-ui-kit';
+import { BodyType } from '../../../types/enum';
+
+export interface IProposalCreationSettingsDefaultProps extends IPluginProposalCreationSettingsParams {}
+
+export const ProposalCreationSettingsDefault: React.FC<IProposalCreationSettingsDefaultProps> = (props) => {
+    const { formPrefix, disableCheckbox } = props;
+    const body = props.body as ISetupBodyForm;
+    const isExternal = body.type === BodyType.EXTERNAL;
+
+    const name = isExternal ? addressUtils.truncateAddress(body.address) : (body.name ?? '');
+    const description = isExternal ? (body.isSafe ? 'Safe{Wallet}' : undefined) : body.description;
+
+    const { value: canCreateProposal, onChange: onCreateProposalChange } = useFormField<
+        ISetupBodyForm,
+        'canCreateProposal'
+    >('canCreateProposal', { fieldPrefix: formPrefix, defaultValue: true });
+
+    const handleCheckedChange = (checked: CheckboxState) => {
+        if (!disableCheckbox) {
+            onCreateProposalChange(checked === true);
+        }
+    };
+
+    return (
+        <CheckboxCard
+            avatar={isExternal && body.isSafe ? safeWallet.src : undefined}
```

Comment:
Maybe it's overkill but if we use this check a lot across the repo, we could maybe centralize on a helper function or utils type check?